api/src/pcapi/routes/pro/collective_offers.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from flask_login import current_user
+from flask_login import login_required
+
+from pcapi.core.educational import api as educational_api
+from pcapi.routes.apis import private_api
+from pcapi.routes.serialization import collective_offers_serialize
+from pcapi.serialization.decorator import spectree_serialize
+
+from . import blueprint
+
+
+logger = logging.getLogger(__name__)
+
+
+@private_api.route("/collective/offers", methods=["GET"])
+@login_required
+@spectree_serialize(response_model=collective_offers_serialize.ListCollectiveOffersResponseModel, api=blueprint.pro_private_schema)  # type: ignore
+def list_collective_offers(
+    query: collective_offers_serialize.ListOffersQueryModel,
+) -> collective_offers_serialize.ListCollectiveOffersResponseModel:
+    paginated_offers = educational_api.list_collective_offers_for_pro_user(
+        user_id=current_user.id,
+        user_is_admin=current_user.has_admin_role,
+        category_id=query.categoryId,
+        offerer_id=query.offerer_id,
+        venue_id=query.venue_id,
+        name_keywords_or_isbn=query.nameOrIsbn,
```

Comment:
Si tu prends le retour dans la serialization:
```suggestion
        name_keywords_or_isbn=query.name,
```

Reply:
je prend a moitier (je ne change pas l'interface rest mais derriere tout est passé a name_keywords